src/robot_dart/control/pd_control.hpp

```diff
@@ -29,6 +31,16 @@ namespace robot_dart {
         protected:
             Eigen::VectorXd _Kp;
             Eigen::VectorXd _Kd;
+
+            inline double angle_dist(const double alfa, const double beta){
```

Comment:
No reason to inline this. Please put the implementation inside the cpp file. Also if you keep it as a member function, it should be prefixed by "_" (-> _angle_dist).